tests/Spec.hs

```diff
@@ -105,6 +107,7 @@ data Results = Results
   , resultGetOpenPullRequests :: [Maybe IntSet]
   , resultGetLatestVersion    :: [Either TagName Integer]
   , resultGetChangelog        :: [Maybe Text]
+  , resultGetDateTime         :: T.UTCTime
```

Comment:
Can you make this a list so we can give concurrent calls to the function increasing timestamps?